CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - GUI programs launched by Alacritty starting in the background on X11
 - Text Cursor position when scrolling
+- The window is now filled with the `background color` before displaying
```

Comment:
This suggests that `background color` is a specific config variable, which it is not. So the graves around it should just be removed.